tests/easier68k/core/opcodes/test_adda.py

```diff
@@ -80,24 +59,15 @@ def test_adda_zero():
 
     adda = Adda(params, OpSize.WORD)
 
-    adda.execute(sim)
+    run_opcode_test(sim, adda, Register.A5, 0b0, 4, [False, False, False, False, False])
```

Comment:
Trivial complaint, but I think you should replace `0b0` with `0x0` or `0` here, so that if you are going to indicate a base, it matches the instruction that is commented. Or a value of 0 will do.